docs/usage/dashboards.md

```diff
@@ -1,21 +1,129 @@
+## Access control
+
+In OpenCTI, the power to create custom dashboards comes with a flexible access control system, allowing users to tailor visibility and rights according to their collaborative needs.
+Access Restrictions:
+
+When a user crafts a personalized dashboard, by default, it remains visible only to the dashboard creator. At this stage, the creator holds administrative access rights. However, they can extend access and rights to others using the "Manage access" button, denoted by a locker icon, located at the top right of the dashboard page.
+
+![Manage access button](assets/manage-access-button.png)
+
+**Levels of access:**
+
+- View: Access to view the dashboard. 
+- Edit: View + Permission to modify and update the dashboard and its widgets. 
+- Manage: Edit + Ability to delete the dashboard and to control user access and rights.
+
+It's crucial to emphasize that at least one user must retain admin access to ensure ongoing management capabilities for the dashboard.
+
+![Manage access dialog](assets/manage-access-dialog.png) 
+
+!!! note "Knowledge access restriction"
+
+    The platform's data access restrictions also apply to dashboards. The data displayed in the widgets is subject to the user's access rights within the platform. Therefore, an admin user will not see the same results in the widgets as a user with limited access, such as viewing only TLP:CLEAR data (assuming the platform contains data beyond TLP:CLEAR). 
+
+
+## Share dashboard and widget configurations
+
+OpenCTI provides functionality for exporting, importing and duplicating dashboard and widget configurations, facilitating the seamless transfer of customized dashboard setups between instances or users.
+
+### Export
+
+Dashboards can be exported from either the custom dashboards list or the dashboard view. 
+
+To export a dashboard configuration from the custom dashboards list:
+
+1. Click on the burger menu button at the end of the dashboard line.
+2. Select `Export`.
+
+![Export dashboard option](assets/export-dashboard-option.png)
+
+To export a widget, the same mechanism is used, but from the burger menu button in the upper right-hand corner of the widget.
+
+![Export widget option](assets/export-widget-option.png)
+
+To export a dashboard configuration from the dashboard view:
+
+1. Navigate to the desired dashboard.
+2. Click on the `Export to JSON` button (file with an arrow pointing to the top right corner) located in the top-right corner of the dashboard.
+
+![Export JSON button](assets/export-json-button.png)
+
+The configuration will be saved as a JSON file, with the title formatted as `[YYYYMMDD]_octi_dashboard_[dashboard title]`. The expected configuration file content is as follows:
+
+```JSON
+{
+  "openCTI_version": "5.12.0",
+  "type": "dashboard",
+  "configuration": {
+    "manifest": "eyJ3aWRn(...)bmZpZyI6e319",
+    "name": "Hello Dashboard From Import"
+  }
+}
+```
+
+!!! warning "Source platform-related filters"
```

Comment:
I don't get what you mean

Reply:
For example, if you have the filter "Label:my-special-label" on the source platform (i.e. where you export the dashboard) and you don't have this label on the receiving platform (i.e. where you import the dashboard)

Do you have a rephrasing suggestion to help people understand?

Reply:
Ok! I see. I think it's clear the way you said it. I just did not get it